src/kvilib/core/KviMemory.cpp

```diff
@@ -113,7 +113,7 @@ namespace KviMemory
 			}
 			e = e->next;
 		}
-		fprintf(stderr,"Attempt to realloc an inexistant pointer (%p) (called from %p (%p))\n",ptr,__builtin_return_address(1),__builtin_return_address(2));
+		fprintf(stderr,"Attempt to realloc an non-existent pointer (%p) (called from %p (%p))\n",ptr,__builtin_return_address(1),__builtin_return_address(2));
```

Comment:
`Attempt to realloc a non-existent pointer` You only use `an` when the follow word begins with a vowel. e.g. `An error has occured` v.s. `A connection is already in progress`